challenge2-incremental/incremental_client.py

```diff
@@ -32,8 +31,8 @@ def ordered_dict_to_json(ordered_dict):
     return json.dumps(model_state_dict_serializable)
 
 def get_initial_server_data():
-    # response = requests.get(server_location+"/get_initial_server_data")
-    return './models/global_parameters.pt'
+    response = requests.get(server_location+"/get_initial_server_data")
+    return json_to_ordered_dict(response.json())
```

Comment:
This is an endpoint that grabs the initial weights(ordered dict) from the local pt file from the server

Reply:
Yes this makes sense.